packages/jest-cli/src/cli/index.js

```diff
@@ -263,7 +263,6 @@ const _run = async (
 
   argv.watch || argv.watchAll
```

Comment:
oh wow... i didn't notice this. This simplifies things even further. All of those functions don't need `argv` any more. (and flow didn't catch that again) :)